### PR TITLE
Change the administrator binding label to something more appropriate

### PR DESCRIPTION
**Pull request by VOID404**
**Description**

I used `reconciler.kyma-project.io/managed-by=reconciler` as suggested in the original Issue (#1912).
I kept the `type=admin`, so we don't remove some other reconciler managed resources.
For now all Cluster Role Bindings with `type=admin` will still be removed, this has to change after clusters are upgraded.

Changes proposed in this pull request:

- Add `reconciler.kyma-project.io/managed-by=reconciler` to Cluster Role Bindings for admins
- Add filter to remove Cluster Role Bindings with new label

**Related issue(s)**

#1912

### CODE REVIEW

File: components/provisioner/internal/operations/stages/provisioning/create_operators_bindings.go [hunk 1]
```diff
@@ -128,17 +129,25 @@ func (s *CreateBindingsForOperatorsStep) Run(cluster model.Cluster, _ model.Oper
...
+						"type":                                  "admin",
+					}))
 		}
 	}
 
 	if err := createClusterRoles(k8sClient.RbacV1().ClusterRoles(), clusterRoles); err != nil {
 		return operations.StageResult{}, err
 	}
 
+	//TODO: remove me after all clusters migrated
```
Comment (franpog859) [inline components/provisioner/internal/operations/stages/provisioning/create_operators_bindings.go:144]: I'd not add a TODO in the code :d It ends up living there for years. It's better to make a follow-up issue

**Review by franpog859 — COMMENTED** [review]
I think I'd add these labels to the L2 and L3 ClusterRoleBindings too

File: components/provisioner/internal/operations/stages/provisioning/create_operators_bindings.go (hunk 1 above)
Comment (VOID404) [reply components/provisioner/internal/operations/stages/provisioning/create_operators_bindings.go]: Fixed

**Comment by VOID404** [comment]
Moved to #2209

**Pull request closed without merging**